lib/internal/modules/cjs/loader.js

```diff
@@ -85,6 +85,27 @@ const relativeResolveCache = Object.create(null);
 
 let requireDepth = 0;
 let statCache = null;
+
+function enrichCJSError(err) {
+  const stack = err.stack.split('\n');
+
+  const lineWithErr = stack.findIndex((line) =>
```

Comment:
Is this line not always the same number?

Reply:
yes its index `1` i removed `findIndex`